Update PARSINGERROR.md

GaucovyAero opened the pull request:
gamink, should fix the stupid double page thingus (idk how to explain the issue, will attach a screenshot if possible)

![image](https://user-images.githubusercontent.com/85581164/219974920-734d8ffd-2c42-4a36-9f3a-626b17906220.png)

Doregon commented:
Merged, although I didn't have the time to merge correctly so I just made a new commit on the main branch.

GaucovyAero committed:
Update PARSINGERROR.md

gamink, should fix the stupid double page thingus (idk how to explain the issue, will attach a screenshot if possible)

Signed-off-by: GaucovyAero <85581164+GaucovyAero@users.noreply.github.com>

mchnjt reviewed (approved):
.
